src/cpufreq.c

```diff
@@ -28,18 +31,52 @@
 
 static int num_cpu = 0;
 
+static char const * freq_fname = "/sys/devices/system/cpu/cpu%d/cpufreq/"
+								 "scaling_cur_freq";
+static char const * freq_fname_rhel = "/sys/devices/system/cpu/cpu%d/cpufreq/"
```

Comment:
I saw came across with the problem first time on CentOS 7.1, that's why used rhel suffix.
Agree with you, but don't have enough time to dig when and why the path was changed.
So, what about "_alt" (i.e. alternative) suffix? Or any other suggestions?